lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_tools_path_helper.rb

```diff
@@ -10,7 +10,7 @@ class ToolsPathHelper
         attr_reader :android_home
 
         def initialize(sdk_root: nil)
-          @android_home = sdk_root || ENV['ANDROID_HOME'] || ENV['ANDROID_SDK_ROOT'] || ENV['ANDROID_SDK']
+          @android_home = sdk_root || ENV.fetch('ANDROID_HOME', nil) || ENV.fetch('ANDROID_SDK_ROOT', nil) || ENV.fetch('ANDROID_SDK', nil)
```

Comment:
Rubocop's automatic change here was 

`@android_home = sdk_root || ENV['ANDROID_HOME'] || ENV['ANDROID_SDK_ROOT'] || ENV.fetch('ANDROID_SDK', nil)` 

I wasn't sure if it just missed the first two `ENV[]` instances, so I manually added `ENV.fetch` to those two. Was that the correct thing to do?

Reply:
Yeah seems like a strange change from rubocop. Especially since `ENV.fetch(key, nil)` is exactly the same as `ENV[key]` (see [doc](https://apidock.com/ruby/Hash/fetch))…

If it were me I'd just keep `ENV[…]` for all the cases in that line. But rubocop will then likely complain so…
Another option is to rewrite this using something like:

```ruby
@android_home = sdk_root || ENV.values_at('ANDROID_HOME', 'ANDROID_SDK_ROOT', 'ANDROID_SDK'].compact.first
```

But I'm not sure that better… 🤷

Reply:
Same as https://github.com/wordpress-mobile/release-toolkit/pull/464#discussion_r1175891328 and the other instances of `ENV.fetch` in this PR. I think that may be a better approach than needing to add a new Rubocop exception.